spec/system/admin/orders_spec.rb

```diff
@@ -223,12 +223,31 @@
             login_as_admin_and_visit spree.admin_orders_path
           end
           it "orders by completion date" do
-            find("a", :text => 'COMPLETED AT').click # sets ascending ordering
+            find("a", text: 'COMPLETED AT').click # sets ascending ordering
```

Comment:
I prefer fixes like this to be squashed before marking a PR as ready for review. And it should be separate from other changes like adding specs.